fiber/middleware.go

```diff
@@ -0,0 +1,21 @@
+package fiber
+
+import (
+	"context"
+
+	"github.com/airbrake/gobrake/v5"
+
+	"github.com/gofiber/fiber/v2"
+)
+
+// New returns a function that satisfies fiber.Handler interface
+func New(notifier *gobrake.Notifier) fiber.Handler {
+	return func(c *fiber.Ctx) error {
+		err := c.Next()
+		_, metric := gobrake.NewRouteMetric(context.TODO(), c.Route().Method, c.Route().Path)
+
+		metric.StatusCode = c.Response().StatusCode()
+		_ = notifier.Routes.Notify(context.TODO(), metric)
```

Comment:
What if notifier is nil?

Reply:
The go compiler will throw error as it is expecting a non-nil entity.

Reply:
My understanding is that this would result in a runtime panic if `notifier` is nil. The `notifier` parameter is a pointer in the method signature, which means that a nil could be passed into this method and the compiler wouldn't catch that at compile time.

Reply:
@ChimanJain this comment has not been addressed yet. Perhaps creating a test for this function and passing nil, `fiber.New(nil)` can show the behavior.